src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -95,7 +99,7 @@ studio.configuration.site.roleMappingsFileName: role-mappings-config.xml
 # Defines name used for environment specific configuration. It is used for environment overrides in studio. Default value is local.
 studio.configuration.site.environment: local
 # File name where environment specific configuration is stored
-studio.configuration.site.environment.configFileName: environment-config.xml
+studio.configuration.site.environment.configFileName: environment/environment-config.xml
```

Comment:
These are big changes, are you adding handlers in the upgrade manager?

Reply:
upgrade manager takes care of all config pattern changes. this one is handled by patterns above (previous comments).